checklists/network_appdelivery_checklist.en.json

```diff
@@ -197,6 +197,7 @@
             "ammp": true,
             "severity": "High",
             "link": "https://learn.microsoft.com/azure/web-application-firewall/ag/application-gateway-waf-request-size-limits#request-body-inspection"
+            "graph": "resources | where type == 'microsoft.network/applicationgatewaywebapplicationfirewallpolicies' | where properties['policySettings']['requestBodyCheck'] != "null" | where properties['policySettings']['state'] == "Enabled" | distinct id, name"
```

Comment:
You should probably use single quotes inside of the `graph` element for strings: `'null'` instead of `"null"`. Besides, are we sure that the null value is stored as string?